x/campaign/client/cli/query_maximum_shares_test.go

```diff
@@ -0,0 +1,48 @@
+package cli_test
```

Comment:
I suggest removing this test as it doesn't bring value above `grpc_maximum_shares_test.go` and increase the likelihood to have the `address already in use` error for action